include/fmt/color.h

```diff
@@ -185,11 +185,17 @@ enum class terminal_color : uint8_t {
 
 enum class emphasis : uint8_t {
   bold = 1,
-  italic = 1 << 1,
-  underline = 1 << 2,
-  strikethrough = 1 << 3
+  faint = 1 << 1,
+  italic = 1 << 2,
+  underline = 1 << 3,
+  blink = 1 << 4,
+  reverse = 1 << 5,
+  conceal = 1 << 6,
+  strikethrough = 1 << 7,
 };
 
+constexpr size_t num_emphases = 8;
```

Comment:
It's an internal symbol only used in `rgb` so let's move it there and make private (and static).

Reply:
Sorry, I don't understand how it is only used in the `rgb` struct; shouldn't it be in the `ansi_color_escape` struct only ?